src/main.js

```diff
@@ -1,9 +1,17 @@
+// vue-virtual-scroller needs this polyfill: https://github.com/Akryum/vue-virtual-scroller#installation
+import "intersection-observer";
+
 import Vue from "vue";
+import VueVirtualScroller from "vue-virtual-scroller";
+import "vue-virtual-scroller/dist/vue-virtual-scroller.css";
+
 import App from "@/App.vue";
 import router from "@/router";
 import "@/config";
 import store from "./store/store";
 
+Vue.use(VueVirtualScroller);
```

Comment:
Could you please create a `virtual-scroller.js` file within the config folder with all the configuration necessary for this?

Then import it in `config.js`

Reply:
Sure, will be doing that.

Reply:
Done.